src/applications/beta-enrollment/containers/BetaEnrollmentButton.jsx

```diff
@@ -49,6 +49,10 @@ class BetaEnrollmentButton extends React.Component {
       );
     }
 
+    if (this.props.renderButton) {
```

Comment:
This seemed like the easiest way to put a check in front of the button for our submission limit.